metadata-ingestion/src/datahub/ingestion/extractor/protobuf_util.py

```diff
@@ -65,17 +65,19 @@ def protobuf_schema_to_mce_fields(
     main_schema: ProtobufSchema,
     imported_schemas: Optional[List[ProtobufSchema]] = None,
     is_key_schema: bool = False,
+    message_name: str = None
 ) -> List[SchemaField]:
     """
     Converts a protobuf schema into a schema compatible with MCE
+    :param message_name: message name
```

Comment:
Real Nit Picky: Is there a better description available here than just "message name"? I'm not saying this has to change but just was curious

Reply:
Added more details